jaxrs/pom.xml

```diff
@@ -111,25 +111,30 @@
         </dependency>
 
         <dependency>
-            <groupId>org.jboss.spec.javax.ws.rs</groupId>
-            <artifactId>jboss-jaxrs-api_2.1_spec</artifactId>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-crypto</artifactId>
         </dependency>
 
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
-            <artifactId>resteasy-crypto</artifactId>
+            <artifactId>resteasy-core</artifactId>
         </dependency>
 
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
-            <artifactId>resteasy-jaxrs</artifactId>
+            <artifactId>resteasy-core-spi</artifactId>
         </dependency>
 
         <dependency>
             <groupId>org.jboss.spec.javax.annotation</groupId>
             <artifactId>jboss-annotations-api_1.3_spec</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
this is probably not so important as both `resteasy-core` and `resteasy-core-spi` depend on `jboss-jaxrs-api_2.1_spec` anyway, just out of curiosity - is there any point in moving this dependency after the `resteasy-core` dependency?

Reply:
No reason other than putting them in alphabetical order.

Reply:
ok. Order matters in maven but this one is probably safe